Tests/AppKitTests/NSColorExtensionsTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UIColorExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Max Haertwig on 10/08/19.
+//  Copyright © 2019 SwifterSwift
+//
+
+import Foundation
```

Comment:
Shouldn't it be the same as the source file?
```Swift
#if canImport(Cocoa)
import Cocoa
```

Reply:
Of course